CMakeLists.txt

```diff
@@ -72,9 +72,9 @@ target_include_directories(${PROJECT_NAME}
         ${EIGEN3_INCLUDE_DIRS}
 )
 if (WITH_PCL)
-    target_include_directories(${PROJECT_NAME} PUBLIC ${PCL_INCLUDE_DIRS})
```

Comment:
but this would be wrong. client projects need to include pcl include dirs if they want to make use of `pcl.hpp` shipped with this project.
same with libs

Reply:
See the description of the PR, looks like it still produces the correct includes. Thing is we just cannot use absolute paths to where the includes "were" during install if it should be relocatable.

Reply:
where do you see the correct pcl dirs being included? did you try with pcl being in a different prefix than this library?

Reply:
Yea its not in there, but if it contains a path which does not exist anymore, it errors.

Reply:
yes no doubt, but I'm not sure what the fix is.